pkgs/data/themes/obsidian2/default.nix

```diff
@@ -2,21 +2,17 @@
 
 stdenv.mkDerivation rec {
   name = "theme-obsidian2-${version}";
-  version = "2.8";
+  version = "2.9";
 
   src = fetchFromGitHub {
     owner = "madmaxms";
     repo = "theme-obsidian-2";
     rev = "v${version}";
-    sha256 = "0qryqpyxbhr0kyar2cshwhzv4da6rfz9gi4wjb6xvcb6szxhlcaq";
+    sha256 = "1m89ws2a4nms4m8187d5cxi281b66i59xa5shlp3g1r2jc4312cy";
   };
 
   propagatedUserEnvPkgs = [ gtk-engine-murrine ];
 
-  postPatch = ''
-    sed -i -e 's|Obsidian-2-Local|Obsidian-2|' Obsidian-2/index.theme
-  '';
-
   installPhase = ''
```

Comment:
Please add ```runHook preInstall``` (and ```runHook postInstall``` at the end) when overriding the whole phase.

Reply:
Done.